analyzer/api.go

```diff
@@ -81,100 +70,3 @@ type RoundRange struct {
 	// To is the last block to process in this range, inclusive.
 	To uint64
 }
-
-// ChainID is the ID of a chain.
-type ChainID string
-
-// String returns the string representation of a ChainID.
-func (c ChainID) String() string {
-	return string(c)
-}
-
-// Network is an instance of the Oasis Network.
-type Network uint8
-
-const (
-	// NetworkTestnet is the identifier for testnet.
-	NetworkTestnet Network = iota
-	// NetworkMainnet is the identifier for mainnet.
-	NetworkMainnet
-	// NetworkUnknown is the identifier for an unknown network.
-	NetworkUnknown = 255
-)
-
-// FromChainContext identifies a Network using its ChainContext.
-func FromChainContext(chainContext string) (Network, error) {
-	// TODO: Remove this hardcoded value once indexer config supports multiple nodes.
-	if chainContext == "53852332637bacb61b91b6411ab4095168ba02a50be4c3f82448438826f23898" {
-		return NetworkMainnet, nil // cobalt mainnet
-	}
-	var network Network
-	for name, nw := range oasisConfig.DefaultNetworks.All {
-		if nw.ChainContext == chainContext {
-			if err := network.Set(name); err != nil {
-				return NetworkUnknown, err
-			}
-			return network, nil
-		}
-	}
-
-	return NetworkUnknown, ErrNetworkUnknown
-}
-
-// Set sets the Network to the value specified by the provided string.
-func (n *Network) Set(s string) error {
-	switch strings.ToLower(s) {
-	case "mainnet":
-		*n = NetworkMainnet
-	case "testnet":
-		*n = NetworkTestnet
-	default:
-		return ErrNetworkUnknown
-	}
-
-	return nil
-}
-
-// String returns the string representation of a network.
-func (n Network) String() string {
-	switch n {
-	case NetworkTestnet:
-		return "testnet"
-	case NetworkMainnet:
-		return "mainnet"
-	default:
-		return "unknown"
-	}
-}
-
-// Runtime is an identifier for a runtime on the Oasis Network.
-type Runtime string
```

Comment:
shouldn't need to import analyzer just to be able to talk about a runtime. moving to common